File: languages/java/src/main/java/com/bloock/sdk/entity/DecrypterArgs.java
package com.bloock.sdk.entity;

import com.bloock.sdk.bridge.proto.RecordOuterClass;

public class DecrypterArgs {
  String key;

  DecrypterArgs(String key) {
    this.key = key;
  }

  RecordOuterClass.DecrypterArgs toProto() {
    return RecordOuterClass.DecrypterArgs.newBuilder().setKey(this.key).build();
  }
}

File: languages/php/Bloock/Entity/DecrypterArgs.php
<?php

namespace Bloock\Entity;

class DecrypterArgs
{
    public $key;

    public function __construct(string $key) {
        $this->key = $key;
    }

    public function toProto(): \Bloock\DecrypterArgs {
        $p = new \Bloock\DecrypterArgs();
        $p->setKey($this->key);
        return $p;
    }
}